test/runtests.jl

```diff
@@ -12,41 +16,42 @@ end
 @safetestset "Integrated LSM unit tests" begin
     include("integrated/soil_canopy_lsm.jl")
 end
-@safetestset "Integrated pond/soil LSM tests" begin
+gpu_broken || @safetestset "Integrated pond/soil LSM tests" begin
```

Comment:
the tests with this `gpu_broken` tag are the ones that fail on GPUs, right?

Reply:
Yes, correct